project1/src/components/Article/ArticleMeta.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import moment from "moment";
+import FollowButton from "../Buttons/FollowButton";
+import FavoriteButtonSlug from "../Buttons/FavoriteButtonSlug";
+import ArticleInfo from "./ArticleInfo";
+
+interface ArticleMetaProps {
+  image: string;
+  username: string;
+  createdAt: string;
+  following: boolean;
+  profileName: string;
+  favorited: boolean;
+  favoritesCount: number;
+  handleFollow: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  handleFavorite: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+}
+
+class ArticleMeta extends React.Component<ArticleMetaProps> {
+  //   handleFollow = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+  //     this.props.handleFollow(event)
+  //   };
+
+  render() {
+    return (
+      <Grid container direction="row" justify="flex-start" alignItems="center">
+        <Grid item xs={4}>
+          <ArticleInfo
+            image={this.props.image}
```

Comment:
optional, but it may be easier to read if you destruct the props at the top of the render function